packages/ffe-searchable-dropdown-react/src/List.js

```diff
@@ -0,0 +1,235 @@
+import {
+    CellMeasurer,
+    CellMeasurerCache,
+    List as VirtualizedList,
+    AutoSizer,
+} from 'react-virtualized';
+import React from 'react';
+import { Scrollbars } from 'react-custom-scrollbars';
+import { Paragraph } from '@sb1/ffe-core-react';
+import {
+    arrayOf,
+    any,
+    bool,
+    func,
+    number,
+    object,
+    oneOf,
+    shape,
+    string,
+} from 'prop-types';
+
+import { getNotMatchText, locales } from './translations';
+import ListItemContainer from './ListItemContainer';
+import { stateChangeTypes } from './reducer';
+
+const NoMatch = ({ noMatch, noMatchMessageId, listToRender }) => (
+    <div>
+        {noMatch.text ? (
+            <div className="ffe-searchable-dropdown__no-match">
+                <Paragraph id={noMatchMessageId}>{noMatch.text}</Paragraph>
+            </div>
+        ) : (
+            listToRender.length === 0 && (
+                <Paragraph
+                    id={noMatchMessageId}
+                    className="ffe-screenreader-only"
+                >
+                    {getNotMatchText()}
+                </Paragraph>
+            )
+        )}
+    </div>
+);
+
+NoMatch.propTypes = {
+    noMatch: shape({
+        text: string,
+        dropdownList: arrayOf(object),
+    }),
+    noMatchMessageId: string,
+    listToRender: arrayOf(object).isRequired,
+};
+
+export default class List extends React.PureComponent {
+    state = {
+        optionHeight: 40,
+    };
+
+    constructor(props, context) {
+        super(props, context);
+
+        this._cache = new CellMeasurerCache({
+            fixedWidth: true,
+            minHeight: 10,
+        });
+
+        this.rowRenderer = this.rowRenderer.bind(this);
+        this.handleScroll = this.handleScroll.bind(this);
+
+        this.listRef = React.createRef();
+    }
+
+    handleScroll = event => {
+        const { scrollTop, scrollLeft } = event.target;
+        const { Grid } = this.listRef.current;
+        Grid.handleScrollEvent({ scrollTop, scrollLeft });
+    };
+
+    componentDidMount() {
+        requestAnimationFrame(() => {
+            const firstOption =
```

Comment:
const firstOption =  this.listRef.current?.Grid?._scrollingContainer?.querySelector(
                    '[role="option"]',
                );